### ISSUE TITLE
Clearly separate Controls, Camera and CameraUtils

### ISSUE THREAD

**Issue by jailln** [iTowns/itowns#945]
Controls, Camera and CameraUtils are not clearly separated and share some responsabilities. Some [GlobeControls](http://www.itowns-project.org/itowns/API_Doc/GlobeControls.html) methods (such as getCameraCoordinate(), getCameraOrientation(), getCameraTargetGeoPosition(), getCameraTargetPosition(), etc.)  are for instance related to the camera.

We should clearly separate these classes with the following responsabilities:
* Camera: simulation of the physical camera
* CameraUtils: utils to move the camera
* Controls: handle user events and uses Camera and CameraUtils to move the Camera when needed

**Comment by gchoqueux** [comment]
Yes indeed, this needs to be clarified. We think to move these methods in the viewer or camera. But before we want only one Viewer (instead of PlanarViewer, GlobeViewer, PanoramaViewer and Viewer) and refactor the Controls.
